notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1282 @@
+{
```

Comment:
<p>Might be good to explain what <code>pagesize</code> does and some of the other parameters. The way it's described here is a good example: https://mast.stsci.edu/api/v0/MastApiTutorial.html </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>